tests/dummy/app/components/ui/form/checkbox.css

```diff
@@ -1,8 +1,3 @@
-.label, .checkbox {
-  margin: 0;
-  padding: 0;
-}
-
```

Comment:
We can remove this block (unnecessary), because we had reset the margin and padding of all elements to `0` in [app.css](https://github.com/ijlee2/ember-container-query/blob/2.0.2/tests/dummy/app/styles/app.css#L16-L19).